submissions/israelOladosu/php/converter.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+function converter ($type, $value = 0) {
+    $onePound =  2.205;
+    $oneYard = 1.094;
```

Comment:
well done @Israel-Force convert `$type` to lower cases such that even when I pass `Kilo` or `KILO`, it will still work the same as `kilo`

Reply:
Thank you. I have done that now.